src/elements/content-sharing/types.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import type { ItemType } from '../../common/types/core';
+import type { item, sharedLinkType } from '../../features/unified-share-modal/flowTypes';
+
+// "SLS" denotes values that are used in the Shared Link Settings modal
+type ContentSharingEnterpriseDataType = {
+    enterpriseName: string,
+    serverURL: string, // SLS
+};
+
+export type ContentSharingUserDataType = {
+    id: string,
+    userEnterpriseData: ContentSharingEnterpriseDataType,
+};
+
+export type ContentSharingSharedLinkType =
+    | { canInvite: boolean }
```

Comment:
Should these type options be aliased to hint at the scenario in which they'd be used?